Fastfile.private

```diff
@@ -58,36 +61,42 @@ reflected in `Info.plist` during building."
   desc "First deploy must always be a '#{:major}'."
   private_lane :release do |options|
 
-    build_configuration_key = options[:build_configuration]
-    build_configuration = get_build_configuration(build_configuration: build_configuration_key)
+    environment = options[:environment]
```

Comment:
I'd keep the `_key` to distinguish between the key name and the actual environment.

Reply:
I'll leave a comment just here, but it's to apply in every place this was changed.

Reply:
I don't get it. Now build_configuration_key is environment and build_configuration is buikld_configuration.
`:environment` is not a variable, just a symbol, like "environment".

Reply:
Yes, sure, ignore my comment.